packages/chart/src/XYAxis.ts

```diff
@@ -194,7 +194,7 @@ export class XYAxis extends SVGWidget {
         const currSel: any = d3BrushSelection(this.svgBrush.node());
         if (currSel) {
             selected = this.data().filter(function (d) {
-                const pos = context.dataPos(d[0]) + context.domainAxis.bandwidth() / 2;
```

Comment:
Why was ` + context.domainAxis.bandwidth() / 2` removed? Should the same thing happen to ` + context.valueAxis.bandwidth() / 2` on line 199?

Reply:
It was not needed (once I started highlighting which ones were selected it was pretty obvious).  and yes probably...

Reply:
Which may mean that when the X-Axis is linear it will be needed

Reply:
Just tested seems to work as expected - the 2D selection is still a bit wonky though (the handles don't move with it and there are no handles on the vertical...